at_secondary/at_persistence_secondary_server/lib/src/log/commitlog/commit_log_keystore.dart

```diff
@@ -258,11 +258,17 @@ class CommitLogKeyStore
 
   @override
   Future<List<dynamic>> getExpired(int expiryInDays) async {
-    final dupEntries = await getDuplicateEntries();
-
-    _logger.finer('commit log entries to delete: $dupEntries');
-
-    return dupEntries;
+    var expiredKeys = <dynamic>[];
+    var now = DateTime.now().toUtc();
+    var commitLogMap = await toMap();
+    commitLogMap.forEach((key, value) {
+      if (value.opTime != null &&
+          value.opTime!
+              .isBefore(now.subtract(Duration(days: expiryInDays)))) {
+        expiredKeys.add(key);
+      }
+    });
+    return expiredKeys;
```

Comment:
How will this method be used?

Reply:
this method will not be used in latest changes..I changed the impl to avoid confusion in implementation old interface -AtLogType. Will revisit this method and remove.